modules/javafx.graphics/src/main/java/javafx/scene/SubScene.java

```diff
@@ -316,6 +316,7 @@ protected void invalidated() {
                     if (oldRoot != null) {
                         StyleManager.getInstance().forget(SubScene.this);
```

Comment:
Not part of the story, but interesting that here the sub scene will be 'forgotten' but nothing like this happens in the `Scene` class. 😄